### PR TITLE
Release: amazon alpha.4

### PR DESCRIPTION
**Pull request by aboedo**
## 4.0.0-amazon.alpha.4

- Fixes a crash when running on iOS
    https://github.com/RevenueCat/purchases-flutter/pull/337

### CODE REVIEW

File: RELEASING-AMAZON.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+1. Start a branch `release/amazon-x.x.x`
+1. Run `flutter format .`
+1. Update CHANGELOG-LATEST.md with the changes for the current version (to be used by Fastlane for the github release notes)
+1. Run `bundle exec fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in pubspec.yaml, purchases_flutter.podspec and android/build.gradle.
+1. `flutter pub pub publish --dry-run`
+1. If purchases-hybrid-common was updated, run `pod update PurchasesHybridCommon` in both `MagicWeather` and `purchase_tester`
+1. Commit the changes, open a PR into main, merge when ready
```
Comment (vegaro) [inline RELEASING-AMAZON.md:7]: 😄

File: RELEASING-AMAZON.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1. `git tag amazon-x.x.x`
+1. `git tag amazon-latest`
+1. `git push origin main && git push --tags`
```
Comment (vegaro) [inline RELEASING-AMAZON.md:10]: oops

File: RELEASING-AMAZON.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+1. Start a branch `release/amazon-x.x.x`
+1. Run `flutter format .`
+1. Update CHANGELOG-LATEST.md with the changes for the current version (to be used by Fastlane for the github release notes)
+1. Run `bundle exec fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in pubspec.yaml, purchases_flutter.podspec and android/build.gradle.
```
Comment (vegaro) [inline RELEASING-AMAZON.md:4]: did you run it for this PR and it worked fine?

File: RELEASING-AMAZON.md (hunk 2 above)
Comment (aboedo) [reply RELEASING-AMAZON.md]: LOL

File: RELEASING-AMAZON.md (hunk 1 above)
Comment (aboedo) [reply RELEASING-AMAZON.md]: I'm going to start a copy / paste jar where I have to put a dollar in whenever I make a copy / paste bug. 
In a year I'll use it to buy a car.

File: RELEASING-AMAZON.md (hunk 3 above)
Comment (aboedo) [reply RELEASING-AMAZON.md:4]: yep! it only updates a handful of files, I didn't run into any issues

**Comment by aboedo** [comment]
integration tests are angry at me and I'm not sure why

**Review by vegaro — APPROVED** [review]

**Comment by vegaro** [comment]
Oh! I don't think I have actually ever run the integration tests in this branch! I don't think I ever did a `release/x.x.x` branch. Like the PR I did the other day was another branch not named `release/x.y.z` https://github.com/RevenueCat/purchases-flutter/pull/336. So I think it's normal they are angry. I can look into them if you want

**Comment by vegaro** [comment]
@aboedo fixed!

**Comment by aboedo** [comment]
@vegaro thanks for taking care of the integration tests!! 🎉

**Pull request merged**